packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -102,6 +118,126 @@ describe("Integration tests", () => {
         expect(1).toBe(1);
       });
     });
+    describe("updateTenantVerifiedAttribute", async () => {
+      const updatedAt = new Date();
+      const mockVerificationAttributeSeed: UpdateVerifiedTenantAttributeSeed = {
+        expirationDate: new Date(
+          new Date().setDate(new Date().getDate() + 1)
+        ).toDateString(),
+      };
+      const expirationDate = new Date(
+        mockVerificationAttributeSeed.expirationDate!
+      );
+
+      const updatedTenant: Tenant = {
+        ...mockTenant,
+        attributes: [
+          {
+            ...mockVerifiedTenantAttribute,
+            verifiedBy: [
+              {
+                ...mockVerifiedBy,
+                expirationDate,
+              },
+            ],
+          },
+        ],
+        updatedAt,
+        name: "A updatedTenant",
+      };
+      const attributeId = updatedTenant.attributes.map((a) => a.id)[0];
+      const verifierId = mockVerifiedBy.id;
+      it("Should update the expirationDate", async () => {
+        vi.useFakeTimers();
+        vi.setSystemTime(updatedAt);
+        await addOneTenant(updatedTenant, postgresDB, tenants);
+        await tenantService.updateTenantVerifiedAttribute({
+          verifierId,
+          tenantId: updatedTenant.id,
+          attributeId,
+          updateVerifiedTenantAttributeSeed: mockVerificationAttributeSeed,
+        });
+        const writtenEvent = await readLastEventByStreamId(
+          updatedTenant.id,
+          postgresDB
+        );
+        expect(writtenEvent.stream_id).toBe(updatedTenant.id);
+        expect(writtenEvent.version).toBe("1");
+        expect(writtenEvent.type).toBe("TenantUpdated");
+        const writtenPayload = decodeProtobufPayload({
+          messageType: TenantUpdatedV1,
+          payload: writtenEvent.data,
+        });
+        expect(writtenPayload.tenant).toEqual(toTenantV1(updatedTenant));
+
+        vi.useRealTimers();
+      });
+      it("Should throw tenantNotFound when tenant doesn't exist", async () => {
+        expect(
+          tenantService.updateTenantVerifiedAttribute({
+            verifierId,
+            tenantId: updatedTenant.id,
+            attributeId,
+            updateVerifiedTenantAttributeSeed: mockVerificationAttributeSeed,
+          })
+        ).rejects.toThrowError(tenantNotFound(updatedTenant.id));
+      });
+
+      it("Should throw expirationDateCannotBeInThePast when expiration date is in the past", async () => {
+        vi.useFakeTimers();
+        vi.setSystemTime(updatedAt);
+        const mockVerificationAttributeSeed: UpdateVerifiedTenantAttributeSeed =
+          {
+            expirationDate: new Date(
+              new Date().setDate(new Date().getDate() - 1)
+            ).toDateString(),
+          };
```

Comment:
```suggestion
```
Since this is needed only in this specific test we can omit the declaration of the whole object, see how we can bypass this with the suggestion on `updateTenantVerifiedAttribute`